hack/make-rules/make-help.sh

```diff
@@ -23,7 +23,7 @@ readonly reset=$(tput sgr0)
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 ALL_TARGETS=$(make -C "${KUBE_ROOT}" PRINT_HELP=y -rpn | sed -n -e '/^$/ { n ; /^[^ .#][^ ]*:/ { s/:.*$// ; p ; } ; }' | sort)
-CMD_TARGETS=$(cd "${KUBE_ROOT}/cmd"; find . -type d -mindepth 1 -maxdepth 1 | cut -c 3-)
```

Comment:
I could have sworn I'd already done this. I _have_ done this in hack/lib/util.sh

Reply:
xref: https://github.com/kubernetes/release/pull/802